### ISSUE TITLE
xUnit test assemblies are running in parallel

### ISSUE THREAD

**Issue by jefflill** [nforgeio/neonKUBE#1147]
We don't really support running unit tests in parallel.  I've always thought adding adding this would disable parallel execution:
```
using Xunit;

// Disable parallel test execution because [TestFixture] doesn't
// support this in general.

[assembly: CollectionBehavior(DisableTestParallelization = true, MaxParallelThreads = 1)]
```
This does disable parallel execution within any given test project, but it **does not** prevent multiple test projects from being run in parallel.  Here's some more details on how this really works:

https://xunit.net/docs/running-tests-in-parallel

It appears the solution is to annotate all of our test classes with something like:
```
[Collection("NonParallel")]
public class TestClass2
{
    [Fact]
    public void Test2()
    {
        Thread.Sleep(5000);
    }
}
```

**Comment by jefflill** [comment]
I've gone through the unit tests and allow some of them (like **Neon.Common**, **Neon.Cryptography**,...) run in parallel and then disabled parallel execution for the others.  This is going to save us maybe 15-20 minutes for execute the entire suite.

**Comment by jefflill** [comment]
**FIXED** for neonLIBRARY v2.11.0
